pkgs/development/compilers/scala/dotty.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchurl, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  version = "0.4.0-RC1";
+  name = "dotty-${version}";
+
+  src = fetchurl {
```

Comment:
Can you use fetchFromGitHub here instead?

Reply:
@disassembler This is a prebuilt release, to make this derivation build completely from source would require a lot of non-trivial work.